src/libraries/System.Runtime.Numerics/src/System/Numerics/BigInteger.cs

```diff
@@ -18,6 +20,9 @@ namespace System.Numerics
         private const int DecimalScaleFactorMask = 0x00FF0000;
         private const int DecimalSignMask = unchecked((int)0x80000000);
 
+        //Used to create Span with single element
+        private static uint s_zero = 0;
```

Comment:
```suggestion
        // Used to create Span with single element
        private static readonly uint s_zero = 0;
```

Reply:
I tried to use `readonly` but compiler settings in the repi forbid this and offer to replace `readonly` field with known value with constant.

Reply:
Ah, right that doesn't work.

Since `s_zero` is used in just one place for creating the span, you could
```c#
private static readonly uint[] s_zero = new uint[1];
```
and use this as return value.

Reply:
I'd much prefer the approach gfoidl suggests than bypassing the language rules around length one spans.